bin/inventory-parser.bash

```diff
@@ -0,0 +1,401 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+here="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=bin/common.bash
+source "${here}/common.bash"
+
+# Cache hosts info
+output_hosts_info(){
+  local filename="$1"
+  # shellcheck disable=SC2016
+  path=$(dirname "${filename}")
+  # md5=$(echo -n "${filename}"| xargs -n1 -I{} bash -c 'echo "$1"; cat "$1"' x {}get_group_hosts | md5sum | awk '{print $1}')
+  md5=$(cat "${path}/terraform.tfstate" "${filename}" | md5sum | awk '{print $1}')
```

Comment:
Remove the comment and check if the state file exists before including it in the checksum.

Reply:
Added, thanks.